action.yml

```
@@ -66,6 +66,25 @@ inputs:
     required: false
     default: 'linux/amd64'
 
+  push_by_digest:
+    description: Push the artifact to the registry without any associated tags.
+    required: false
+    default: false
+
+  setup:
```

```suggestion
  buildx_setup:
```

`setup` is a little generic for my tastes